### ISSUE TITLE
[angler] Doesn't boot up. Stuck on google logo

### ISSUE THREAD

**Issue by aKn1ghtOut** [ubports/ubports-installer#1541]
**UBports Installer `0.6.0-beta` (snap)**
Environment: `Ubuntu Core 20 linux 5.8.0-26-generic x64 NodeJS v12.16.3`
Device: angler
Target OS: Ubuntu Touch
Settings: `undefined`
OPEN-CUTS run: https://ubports.open-cuts.org/run/5faf80849702a600079dc73c
Log: *N/A*

Just do all the steps in UBports to install Ubuntu Touch
The device was initially running lineage os based on android 10. Was flashed to 7.1.2 stock as instructed before starting to install Ubuntu Touch

**Comment by NeoTheThird** [comment]
Duplicate of #1537
